### PR TITLE
Update aws-rds/README.md with instructions for enhanced monitoring setup

### PR DESCRIPTION
**Pull request by trevoroc**

### CODE REVIEW

File: aws-rds/README.md [hunk 1]
```diff
@@ -40,6 +40,85 @@ SignalFx provides built-in dashboards for this service. Examples are shown below
...
+to the Encryption Keys tab on the IAM Management Console. Make sure to add the
+users who will need to have access to the Lambda function, as well as
+appropriate admins.
+
+#### Create the Lambda function
+
+##### Select the blueprint
+
+A new Lambda can be created from the Functions page of the Lambda Management
+Console. Filter the available blueprints by **signalfx**, and select *insert
```
Comment (trevoroc) [inline aws-rds/README.md]: I put the placeholder here until we know what the blueprint name will be on AWS

**Comment by molner** [comment]
Looks pretty good. One minor nit - in the "Enhanced Monitoring" section 'setup' should be one word, not two. (i.e. setup, not set up)

Obviously update the blueprint name here as soon as it is known.

This is a topic with lengthier-than-normal instructions, so I'm going to also add @barbara-sfx to the reviewer list as I think she might have some good recommendations around wordsmithing/flow/streamlining/etc.

**Comment by asuresh4** [comment]
Please remove hard returns. Github renderer looks immune to it. However, the official docs site is not.

**Comment by barbara-sfx** [comment]
Looks pretty good. A few comments:

Consider changing structures like "allows you to" to "lets you"

=============

A couple screenshots would be helpful

=============

Enhanced monitoring must be enabled on each RDS instance for them to report metrics. This can be done from the RDS dashboard by selecting Modify under the dropdown of instance actions. The Enable Enhanced Monitoring section is towards the bottom. ==> consider making active and also putting actions in the order they occur, e.g. : You must enable enhanced monitoring on each RDS instance for which you want to report metrics. To do so, go to the RDS dashboard, display the dropdown for instance actions, and select Modify.

====

A new Lambda can be created from ==> You can create a new Lambda from

(you can look for the rest of the passive constructions yourself)

=====

The only other required field is the Filter Name; it doesn't matter what name you choose. ==> this seems strange. Isn't there some kind of guideline worth providing?

=======

should have appeared ==> will appear  (If you want to say should, then you have to say what to do if it doesn't)

==========

You can either provide the names of the metric groups that you want included or the groups you want excluded. If you create both lists, only the 'include' list will be used. The list should simply be the names of the groups, spelled exactly as they are in the documentation (including capitals) separated by a single space. The key for this environment variable should be either groups for the inclusive group or groups_out for the exclusive group. ==> Reorder sentences (some edits also included below):

You can provide the names of the metric groups that you want either included or excluded. The key for this environment variable is `groups` for the inclusive group or `groups_out` for the exclusive group. The list should simply be the names of the groups, spelled exactly as they are in the documentation (including capitals), separated by a single space.  If you create both lists, only the `groups` list will be used.

**Pull request merged**